src/main/java/org/fundacionjala/coding/omar/CesarEncryption.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * @(#)CesarEncryption.java
+ *
+ * Copyright (c) 2018 Jala Foundation.
+ * Address
+ * All rights reserved.
+ *
+ * This software is the confidential and propietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Sun.
+ */
+package org.fundacionjala.coding.omar;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * @author Omar Limbert Huanca Sanchez - AT-[06].
+ * @version 1.0.
+ */
+public class CesarEncryption {
```

Comment:
@AT-06/at-06 
CesarEncryption ?
The class is doing two kind of encryption.
Split in two classes and apply abstraction with encode and decode abstract methods.